src/styles/card.css

```diff
@@ -0,0 +1,85 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  font-family: Roboto, sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-main);
+}
+
+.card__code {
+  margin-bottom: 17px;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-secondary);
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__reviews-value {
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-main);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  margin-bottom: 16px;
```

Comment:
You should not use both top and bottom margin for element (Checklist #11)

Reply:
Yep, agreed. Need to fix this.